docker/mailer

```diff
@@ -29,7 +29,7 @@ ADD tools/c7n_openstack /src/tools/c7n_openstack
 RUN rm -R tools/c7n_openstack/tests
 
 # Install requested providers
-ARG providers="azure gcp kube openstack"
+ARG providers="gcp kube openstack azure"
```

Comment:
Assuming this is because of the dep dance around the sequential poetry installs?

Reply:
it was manually added to one of the images, to address a previous issue on dependencies for that particular image, I went ahead and just added it to the template generator when regenerating all the templates. it is unclear if its still needed re this explicit ordering.